ddsrouter_core/src/cpp/participant/implementations/rtps/CommonParticipant.cpp

```diff
@@ -62,6 +62,14 @@ CommonParticipant::~CommonParticipant()
     }
 }
 
+void CommonParticipant::init()
+{
+    create_participant_(
```

Comment:
As already discussed internally, we should create the RTPS participant disabled and enable it here (after having set the listener in `create_participant_`). Otherwise we could miss callbacks which won't be recovered and thus bridge creation won't happen.

Creating the participant enabled seems like an alternative solution, but unfortunately this results in a data race.